docs/data-companion/quick-start.md

```diff
@@ -0,0 +1,269 @@
+---
+order: 1
+parent:
+    title: Quick Start
+    order: 2
+---
+
+
+# Creating a Data Companion for CometBFT
+
+## Fetching data
```

Comment:
This title is a bit redundant with the Fetching data title below. I think it can even be removed here and this serves as an intro.

Reply:
fixed